.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+
+  - package-ecosystem: 'npm'
+    directories: '/', '/client', '/server'
```

```suggestion
    directory: '/'
```

We currently have only one `package-lock.json` located in the root directory, so pointing to those 2 additional directories does not have any effect.
Once we switch to monorepo (in #523), we will definitely only need `/`, better change it now.